### PR TITLE
Add: DeepSparse performance numbers

### PR DESCRIPTION
**Pull request by rahul-tuli**

### CODE REVIEW

**Comment by dhuangnm** [comment]
Can we fix the deepsparse version issue in the message "** DeepSparse Performance measured on an AWS C5 instance with 24 cores, batch size 64, and 550 x 550 input with version 1.6 of the DeepSparse Engine." as well?

**Comment by rahul-tuli** [comment]
> Can we fix the deepsparse version issue in the message "** DeepSparse Performance measured on an AWS C5 instance with 24 cores, batch size 64, and 550 x 550 input with version 1.6 of the DeepSparse Engine." as well?

Thanks @dhuangnm made the change on SparseML side

File: examples/dbolya-yolact/README.md [hunk 1]
```diff
@@ -44,9 +44,9 @@ run the following examples.
...
-| Pruned Quantized    | A highly sparse, INT8 model that recovers reasonably close to the baseline model. | zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/pruned82_quant-none | 0.282        | 9.7 MB       | -- img/sec               |
+| Baseline            | The baseline, pretrained model on the COCO dataset.                               | zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/base-none           | 0.288        | 170 MB       | 8.22 img/sec             |
+| Pruned              | A highly sparse, FP32 model that recovers close to the baseline model.            | zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/pruned90-none       | 0.286        | 30.1 MB      | 11.56 img/sec            |
+| Pruned Quantized    | A highly sparse, INT8 model that recovers reasonably close to the baseline model. | zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/pruned82_quant-none | 0.282        | 9.7 MB       | 15.47 img/sec            |
```
Comment (markurtz) [inline examples/dbolya-yolact/README.md:50]: @rahul-tuli need to add in information on how the benchmarks were generated

**Review by markurtz — CHANGES_REQUESTED** [review]

File: examples/dbolya-yolact/README.md (hunk 1 above)
Comment (rahul-tuli) [reply examples/dbolya-yolact/README.md:50]: Hi mark the numbers are old; will regenerate and update diiff accordingly

**Pull request closed without merging**